common-rendering/src/components/liveBlockContainer.tsx

```diff
@@ -141,6 +142,14 @@ const LiveBlockContainer = ({
 				}
 				border-bottom: 1px solid ${neutral[86]};
 
+				&:first-child {
+					margin-top: ${space[3]}px;
```

Comment:
Heads up: I can see from Chromatic that this has created additional spacing on the DCR live blocks too. Was that intentional?

Reply:
No I was just trying to keep the margins on the liveblocks in the same place but I'll just add padding to the container on AR